Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
Ok, so looks like there are these options:

- Explicit ack for setup (suggesting resume supported/unsupported and potentially some other optional setup properties)
- Reject setup if resume not supported by server.
- "Soft error" for resume not supported by server and let client decide whether to continue or disconnect. This is not

IMO, the third option is a good fit and does not preclude false start. Do you think it has other drawbacks?